Labeler failing for forks

The labeler action added on #19 is failing for forks with:

```
2020-06-15T20:23:43.5295034Z ##[group]Run actions/labeler@v2
2020-06-15T20:23:43.5295641Z with:
2020-06-15T20:23:43.5296259Z   repo-token: ***
2020-06-15T20:23:43.5296430Z   configuration-path: .github/labeler.yml
2020-06-15T20:23:43.5296606Z ##[endgroup]
2020-06-15T20:23:44.5155877Z ##[error]HttpError: Resource not accessible by integration
2020-06-15T20:23:44.5159361Z ##[error]Resource not accessible by integration
```

--- a/.github/workflows/labeler.yml
+++ b/.github/workflows/labeler.yml
@@ -5,6 +5,7 @@ on:
 jobs:
   triage:
     runs-on: ubuntu-latest
+    continue-on-error: true
     steps:
     - uses: actions/labeler@v2
       with:

--- a/.github/workflows/labeler.yml
+++ b/.github/workflows/labeler.yml
@@ -1,12 +1,13 @@
 name: "Pull Request Labeler"
 on:
-- pull_request
-
+  schedule:
+    - cron: '*/20 * * * *'
 jobs:
-  triage:
+  labeler:
     runs-on: ubuntu-latest
-    continue-on-error: true
     steps:
-    - uses: actions/labeler@v2
-      with:
-        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+      - uses: ignition-tooling/periodic-labeler@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          LABEL_MAPPINGS_FILE: .github/labeler.yml

--- a/.github/workflows/triage.yml
+++ b/.github/workflows/triage.yml
@@ -8,6 +8,7 @@ jobs:
   assign:
     name: Add ticket to inbox
     runs-on: ubuntu-latest
+    continue-on-error: true
     steps:
       - name: Add ticket to inbox
         uses: technote-space/create-project-card-action@v1

